pkg/credentials/file_aws_credentials.go

```diff
@@ -26,7 +26,7 @@ import (
 	"strings"
 	"time"
 
-	ini "gopkg.in/ini.v1"
+	"github.com/go-ini/ini"
```

Comment:
@harshavardhana FYI, I think this might have used the `gopkg` syntax years ago, perhaps before `go mod` supported Github properly? I looked into it a bit but couldn't see why.

`github.com/go-ini/ini v1.67.0` in go.mod should be an equal replacement